README.md

```diff
@@ -1 +1,54 @@
-﻿# Todo List App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](Screenshottt.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- html
+- css
+- JavaScript
+- Webpack
+
+### Prerequisites
+
+### Setup
```

Comment:
- [ ] I think it would better add methods to get your project locally so as to run it